src/user/user.controller.ts

```diff
@@ -73,8 +68,11 @@ export class UserController {
     return this.userService.update(rut, updateUserDto);
   }
 
-  @Delete(':id')
-  remove(@Param('id') id: string) {
-    return this.userService.remove(+id);
+@Delete('delete/:id')
+  @UseGuards(JwtAuthGuard, RolesGuard)
+  @Roles(Role.ADMIN)
+  async remove(@Param('id') id: number) {
+    return await this.userService.softRemove(id);
```

Comment:
It would be better if you called `this.userService.delete(id)` (as it uses the `softDelete` method) instead of `this.userService.softRemove` (doesn't use `softDelete` method)

Reply:
fixed by https://github.com/alkemyTech/CENCO-MIT-Back-T1/pull/27/commits/0138978e7291fab968cf2c19af4eff9dee9d829b